src/Inventory.java

```diff
@@ -0,0 +1,125 @@
+import com.sun.org.apache.xml.internal.resolver.helpers.PublicId;
+import com.sun.tools.javac.jvm.Items;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+/**
+ * Created by EddyJ on 6/16/16.
+ */
+public class Inventory {
+    private Scanner scanner = new Scanner(System.in);
+    private ArrayList<Item> items = new ArrayList<>();
+
+    public ArrayList<Item> getItems() {
+        return items;
+    }
+
+    public void addItem(Item item) {
+        this.items.add(item);
+    }
+
+    public void setInventoryList(ArrayList<Item> inventoryList) {
+        this.items = inventoryList;
+    }
+
+    public void showItems() {
+        System.out.println("Hello, this is your inventory.");
+        for (Item item : items) {
+            System.out.println(item.getId() + " " + item.getName() + " " + item.getQuantity());
+        }
+    }
+
+    public int showMenu() {
+        int option = 0;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Options: " + System.lineSeparator() + "[1] Create a new Item." + System.lineSeparator() +
+                "[2] Choose item(s) to be sold." + System.lineSeparator() + "[3} Remove an Item" + System.lineSeparator() +
+                "[4] Update an Item's quantity." + System.lineSeparator() + "[5] Update price of an Item." + System.lineSeparator() +
+                "[6] Quit the program.");
+        option = Integer.parseInt(scanner.nextLine());
+        return option;
+    }
+
+    public void addItem() {
+        ArrayList<Item> item = new ArrayList<>();
```

Comment:
`item` is probably not the best name as it is a list of items.  Watch for plurals `items`